src/graphapi/graphapi_tensor.cpp

```diff
@@ -257,7 +258,10 @@ void BackendTensorDescriptor::getAttribute(miopenBackendAttributeName_t attribut
             const auto& dimensions = mDescriptor.getDimensions();
             *elementCount          = dimensions.size();
             std::copy_n(dimensions.begin(),
-                        std::min(*elementCount, requestedElementCount),
+                        // WORKAROUND: building on Windows is failing due to conflicting definitions
+                        // of std::min() between the MSVC standard library and HIP Clang wrappers.
+                        *elementCount < requestedElementCount ? *elementCount
+                                                              : requestedElementCount,
```

Comment:
`*elementCount` and `requestedElementCount` are of different types (signedness), which does not look good from the code quality point of view. So the first thing I recommend is getting of the the uncertainty explicitly, e.g.:
```suggestion
                        std::min(*elementCount, static_cast<size_t>(requestedElementCount)),
```
and see if this resolves the issue. It the issue remains, then workaround is welcome provided that is is properly formalized (please see #1685). Thanks!

Reply:
@apwojcik Forgot to mention: the workaround should only be enabled for Windows.

Reply:
[Notice] @apwojcik well, actually the types are identical. Review comment fixed.